x-pack/plugins/serverless_observability/public/components/side_navigation/index.tsx

```diff
@@ -25,6 +25,7 @@ const navigationTree: NavigationTreeDefinition = {
       title: 'Observability',
       icon: 'logoObservability',
       defaultIsCollapsed: false,
+      breadcrumbStatus: 'hidden',
```

Comment:
Hm 🤔 I see that the current implementation isn't exactly like the design you've shared. 
We planned to always have this "home" breadcrumb first. The "Home" leads to the root of the project (Observability in this case). If we'd also have "Observability" breadcrumb, then it would duplicate "home". More details is [here](https://docs.google.com/document/d/1e5SbDPpySiPeBrjgLJD6Qw6fJyiy34uO2dmGLHlu38E/edit#heading=h.femuqke968zo)
Not sure how close this behavior would be to the "final" or if we're going to change it